File: Maximum Sum LCM - GFG/maximum-sum-lcm.java
//{ Driver Code Starts
//Initial Template for Java
import java.util.*;
import java.io.*;
class GFG {
    public static void main(String args[]) throws IOException {
        BufferedReader read =
            new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(read.readLine());
        while (t-- > 0) {
            int n = Integer.parseInt(read.readLine());
            Solution ob = new Solution();

            System.out.println(ob.maxSumLCM(n));
        }
    }
}
// } Driver Code Ends


//User function Template for Java
class Solution {
    static long maxSumLCM(int n) {
        long ans=n+1;;
        // code here
        for(int i=2;i<=(int)Math.sqrt(n);i++)
        {
            if(n%i==0)
            {
                ans+=i;
                if(i!=n/i) ans+=(n/i);
            }
        }
        return ans;
    }
}

File: Maximum Sum LCM - GFG/maximum-sum-lcm.py
#User function Template for python3
class Solution:
    def maxSumLCM (self, n):
        # code here 
        k=n+1
        for i in range(2,int(n**0.5)+1):
            if(n%i==0):
                k+=i
                if(i!=n//i):
                    k+=n//i
        return k            


#{ 
 # Driver Code Starts
#Initial Template for Python 3
if __name__ == '__main__': 
    t = int (input ())
    for _ in range (t):
        n = int(input())
        
        ob = Solution()
        print(ob.maxSumLCM(n))
# } Driver Code Ends